apps/examples/src/examples/exploded/README.md

```diff
@@ -3,6 +3,7 @@ title: Sublibraries
 component: ./ExplodedExample.tsx
 category: editor-api
 priority: 3
+keywords: [exploded, TldrawEditor, ui, components, utils]
```

Comment:
"exploded" probably isn't needed
likewise, let's remove `TldrawEditor` - it wouldn't fit with keywords